metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/database/SraObjectsImporterThroughDatabase.java

```diff
@@ -133,6 +134,7 @@ public Sample importSample(String accession) {
     protected Analysis extractStudyFromAnalysis(AnalysisType analysisType, Analysis analysis) {
         Study study = importStudyFromAnalysis(analysisType.getSTUDYREF().getAccession());
         analysis.setStudy(study);
+        AnalysisEventHandler.validateReferenceSequenceLink(analysis);
```

Comment:
I don't think a method responsible for linking a study and an analysis is the best place to validate the link between analysis and reference sequence(s). Why not in `importAnalysis`?